abnf/odata-abnf-construction-rules.txt

```diff
@@ -525,6 +525,9 @@ rootExpr = %s"$root/" ( entitySetName   [ collectionNavigationExpr ]
 
 firstMemberExpr = memberExpr
                 / inscopeVariableExpr [ "/" memberExpr ]
+                / "$" request-id "/" memberExpr ; reference to single-valued response of earlier request
+                / "$" request-id collectionNavigationExpr ; reference to collection-valued response of earlier request
+                / "$" request-id

```

Comment:
The longer productions must come first, otherwise the test case "member reference in $batch" fails. It also fails for
```
/ "$" request-id [ "/" memberExpr ]
/ "$" request-id [ collectionNavigationExpr ]
```